### PR TITLE
navbar added, smooth scrolling to html, scroll style changing

### PR DESCRIPTION
**Pull request by idenispankov**

### CODE REVIEW

File: blocks/navbar/navbar.css [hunk 1]
```diff
@@ -11,6 +11,8 @@
   position: fixed;
   width: 1060px;
   align-items: center;
+  margin-left: 100px;
+  margin-right: 100px;
 }
```
Comment (Av1sa) [inline blocks/navbar/navbar.css:17]: I think navbar and header is the same, and the black section on top is called main

File: blocks/navbar/navbar.css [hunk 2]
```diff
@@ -11,6 +11,8 @@
   position: fixed;
   width: 1060px;
```
Comment (Av1sa) [inline blocks/navbar/navbar.css]: With width: 1060px; it's not responsive. Maybe try smth like min-width?

File: blocks/navbar/__logo/navbar__logo.css [hunk 3]
```diff
@@ -8,4 +8,9 @@
   line-height: 14px;
   letter-spacing: -0.04em;
   padding: 17px 0 17px 20px;
```
Comment (Av1sa) [inline blocks/navbar/__logo/navbar__logo.css]: You should apply padding: 17px 20px; to the navbar, not to the elements inside

File: blocks/navbar/__logo/navbar__logo.css [hunk 4]
```diff
@@ -8,4 +8,9 @@
...
+  transition: opacity 0.3s linear;
+}
+
+.navbar__logo:hover {
+  opacity: 0.7;
 }
```
Comment (Av1sa) [inline blocks/navbar/__logo/navbar__logo.css:15]: All effects should be in separate classes, so we could apply them to links, buttons etc. Now you have repetitive code.

File: blocks/navbar/__list/-item/navbar__list-item.css [hunk 5]
```diff
@@ -0,0 +1,11 @@
+.navbar__list-item {
+  color: #fff;
+  text-decoration: none;
+  margin-right: 14px;
+  font-style: normal;
```
Comment (Av1sa) [inline blocks/navbar/__list/-item/navbar__list-item.css]: font-style: normal; is a common style, applied in page class, not needed here and in all classes here

File: blocks/navbar/__logo/navbar__logo.css [hunk 6]
```diff
@@ -0,0 +1,11 @@
+.navbar__logo {
+  text-decoration: none;
+  color: #fff;
+  margin: 0;
+  font-weight: 700;
```
Comment (Av1sa) [inline blocks/navbar/__logo/navbar__logo.css]: font-weight: bold; according to the design

File: blocks/navbar/__list/-item/navbar__list-item.css (hunk 5 above)
Comment (Av1sa) [reply blocks/navbar/__list/-item/navbar__list-item.css]: Fix folder names, no hyphen allowed

File: blocks/navbar/__list/-item/navbar__list-item.css [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  letter-spacing: -0.04em;
+  cursor: pointer;
```
Comment (Av1sa) [inline blocks/navbar/__list/-item/navbar__list-item.css]: cursor: pointer; should be in the hover effect

File: index.html [hunk 8]
```diff
@@ -1,118 +1,132 @@
...
+
+        <li>
+          <a class="navbar__list-item" href="#team">Team</a>
+        </li>
+
+        <li>
+          <a class="navbar__list-item" href="#contact">Contact Us</a>
+        </li>
+      </ul>
+      <a class="navbar__social" href="https://www.linkedin.com/company/antrobotics/" target="_blank">In</a>
```
Comment (Av1sa) [inline index.html]: Should be "in", not "In"

File: blocks/navbar/__list/navbar__list.css [hunk 9]
```diff
@@ -0,0 +1,11 @@
+@import url("./-item/navbar__list-item.css");
+
+.navbar__list {
+  display: flex;
+  list-style: none;
+  font-size: 12px;
+  line-height: 14px;
```
Comment (Av1sa) [inline blocks/navbar/__list/navbar__list.css]: font-size: 12px;
line-height: 14px;
Repetitive code in navbar__list-item.css and navbar__list.css

File: blocks/navbar/navbar.css (hunk 1 above)
Comment (idenispankov) [reply blocks/navbar/navbar.css:17]: added padding to header

File: blocks/navbar/navbar.css (hunk 2 above)
Comment (idenispankov) [reply blocks/navbar/navbar.css]: made it min-width

File: blocks/navbar/__logo/navbar__logo.css (hunk 3 above)
Comment (idenispankov) [reply blocks/navbar/__logo/navbar__logo.css]: fixed

File: blocks/navbar/__logo/navbar__logo.css (hunk 4 above)
Comment (idenispankov) [reply blocks/navbar/__logo/navbar__logo.css:15]: create separate class like item__effect and item__effect:hover ?

File: blocks/navbar/__list/-item/navbar__list-item.css (hunk 5 above)
Comment (idenispankov) [reply blocks/navbar/__list/-item/navbar__list-item.css]: fixed

File: blocks/navbar/__logo/navbar__logo.css (hunk 6 above)
Comment (idenispankov) [reply blocks/navbar/__logo/navbar__logo.css]: fixed

File: blocks/navbar/__list/-item/navbar__list-item.css (hunk 7 above)
Comment (idenispankov) [reply blocks/navbar/__list/-item/navbar__list-item.css]: fixed

File: index.html (hunk 8 above)
Comment (idenispankov) [reply index.html]: fixed

File: blocks/navbar/__list/navbar__list.css (hunk 9 above)
Comment (idenispankov) [reply blocks/navbar/__list/navbar__list.css]: fixed, same question to separate class for hover effect

File: blocks/navbar/__logo/navbar__logo.css (hunk 4 above)
Comment (Av1sa) [reply blocks/navbar/__logo/navbar__logo.css:15]: Let's add effects after everything is ready, it will be clearer what we need

**Pull request merged**